pkg/bundle/finalize.go

```diff
@@ -45,7 +45,7 @@ func Finalize(bundlePath, kabManifestPath string) error {
 	}
 
 	mfst.Images = map[string]bundle.Image{}
-	registryClient := registry.NewRegistryClient()
+	registryClient := ggcr.NewRegistryClient()
```

Comment:
This was moved in the following commit https://github.com/pivotal/image-relocation/commit/82d942d97499e63711ec1e8b94e4655364c0a53d#diff-e3cc3dc2bbc95c91bf4c92e17777de9e

I guess more generally @glyn what other packages use this `Finalize` method?

Reply:
> I guess more generally @glyn what other packages use this `Finalize` method?

The method is used by some proprietary code in PFS.